exept/exeptions.py

```diff
@@ -83,4 +83,28 @@ class UserAlreadyExistError(CustomQSportException):
     def __init__(self, date_value):
         self.date_value = date_value
         message = f"User with such data already exist'"
+        super().__init__(message)
+
+
+class NoUsersInDBError(CustomQSportException):
+    status_code = 404
+
+    def __init__(self):
+        message = f"There are no users in database"
+        super().__init__(message)
+
+
+class EmptyRecommendationListForUserError(CustomQSportException):
+    status_code = 204
+
+    def __init__(self, user_id):
+        message = f"There are no recommendations for user '{user_id} at this moment'"
+        super().__init__(message)
+
+
+class NoJWTInCookieError(CustomQSportException):
```

Comment:
That's basically "UserNotLoggedIn" or smth

We don't care what happened to the JWT - if it's corrupted/missing/fake - User is not logged in and may as well go fuck themself